packages/resolve-scripts/src/template/README.md

```diff
@@ -213,10 +214,23 @@ Function that takes initialState (an object returned by the [initialState functi
 
 Root component to be rendered on the server side.
 
-#### events
+#### initialEvents
 *Object*
 
-List of events.
+Initial list of events, which should be transmitted into client side after SPA page has been loaded.
```

Comment:
I don't like the name initialEvents... initialSubscribedEvents or something like that to avoid misunderstanding. And it is better to describe the action that allows subscribing on new events and how it works.